system/scripts/zone/core/calc_combat.cs

```diff
@@ -647,7 +647,7 @@ public float SCR_GetCritChance(ICombatEntity attacker, ICombatEntity target, Ski
 			return 100;
 
 		var critDodgeRate = target.Properties.GetFloat(PropertyName.CRTDR);
-		var critHitRate = attacker.Properties.GetFloat(PropertyName.CRTHR) + modifier.BonusCritChance;
+		var critHitRate = attacker.Properties.GetFloat(PropertyName.CRTHR) * modifier.CritRateMultiplier + modifier.BonusCritChance;
```

Comment:
Just a quick note here but multiplying the CRTHR by 2 (for Annihilation ability bonus) is not the same as multiplying the critChance variable by 2 due to the exponential formula it has.

Example:
critHitRate = 200
critDodgeRate = 50

No Annihilation Buff:
(200 - 50) ^ 0.6 = 20.2% crit chance

Annihilation Buff (2x Crit Chance):
(400 - 50) ^ 0.6 = 33.6% crit chance

Reply:
You're correct, this probably also applies to BonusCritChance as well.